samples/client/petstore/scala-httpclient/pom.xml

```diff
@@ -157,6 +157,28 @@
                     </jvmArgs>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>exec-maven-plugin</artifactId>
+                <version>1.5.0</version>
+                <executions>
+                    <execution>
+                        <id>sbt-test</id>
+                        <phase>integration-test</phase>
+                        <goals>
+                            <goal>exec</goal>
+                        </goals>
+                        <configuration>
+                            <executable>sbt</executable>
+                            <arguments>
+                                <argument>-ivy</argument>
+                                <argument>/home/circleci/.ivy2</argument>
```

Comment:
This won't fail when we run samples locally? I think these could be parameterized as a Maven property as well.

Reply:
Good point. Let me try `~` instead. I remember it won't expand ~ to home directory properly but let's give it another try.

Reply:
Done via https://github.com/OpenAPITools/openapi-generator/pull/6978/commits/aa5fcd8b4a93526c8b5245d3edde79fb42e59f01. Seems to work.

Reply:
UPDATE: it turns out `~` was not resolved properly. I've filed https://github.com/OpenAPITools/openapi-generator/pull/6985 to use `${user.home}` instead